includes/class-wc-calypso-bridge-setup.php

```diff
@@ -187,11 +187,27 @@ public function woocommerce_create_pages_callback() {
 			}
 
 			try {
+				/*
+				 * Force delete all WooCommerce pages. Some themes create them, and we end up with duplicates.
+				 *
+				 * `My Account` page, has slug `my-account`.
+				 * @see WC_Install::create_pages()
+				 */
+				foreach ( [ 'shop', 'cart', 'my-account', 'checkout', 'refund_returns' ] as $page ) {
+					$page = get_page_by_path( $page, ARRAY_A );
+					if ( $page ) {
```

Comment:
Looks good and tested well! I would suggest being a bit more defensive here. Something along these lines:

```suggestion
					if ( is_array( $page ) && isset( $page['ID'] ) ) {
```